### ISSUE TITLE
`set`-related false positive rejection.

### ISSUE THREAD

**Issue by modulovalue** [UserNobody14/tree-sitter-dart#58]
```dart
void main() {
  final set = 0;
  set;
}
```

The grammar appears to reject this program, but a single identifier named "set" is a valid expression.

**Comment by TimWhiting** [comment]
Yeah, set / get has always been tricky. I really don't have the time to look into this, but feel free to make a PR, I'm pretty good at getting to those.
